ocaml/testsuite/tests/typing-layouts-float64/unboxed_floats.ml

```diff
@@ -10,16 +10,13 @@
    ** bytecode
      flags = "-extension layouts_beta"
    ** native
-     flags = "-extension layouts"
    ** bytecode
-     flags = "-extension layouts"
    ** setup-ocamlc.byte-build-env
      ocamlc_byte_exit_status = "2"
    *** ocamlc.byte
+     flags = "-disable-all-extensions"
```

Comment:
Can we also test with `-only-erasable-extensions`? That's what's used in public release, and should stop layouts from working there.

Reply:
The layouts extension is erasable.

(I don't actually know if we have the tooling in place to do the erasing, but if not, I think that bug is independent from the change in this PR.)

Reply:
That sounds bad. I agree it's separate from this PR. But I'll raise the issue elsewhere.